tests/modules/images/webp-uploads/load-tests.php

```diff
@@ -311,6 +311,43 @@ public function it_should_remove_full_size_images_when_no_size_image_exists() {
 		$this->assertFileDoesNotExist( path_join( $dirname, $metadata['sources']['image/jpeg']['file'] ) );
 	}
 
+	/**
+	 * Remove the attached WebP version if the attachment is force deleted but empty trash day is not defined
```

Comment:
What do you mean by "empty trash day is not defined", would the test look different if that was not the case?

Reply:
That was mistakenly added in the function comment. I have fixed the comment now.